### PR TITLE
[System.Web.*] Reference source import

### PR DESCRIPTION
**Pull request by kasthack**
```
[System.Web]
System.Web.
	Profile.
		ProfileInfoCollection
		ProfileInfo
		ProfileAuthenticationOption
		DefaultProfile
	UI.
		ICheckBoxControl
		IEditableTextControl
		ITextControl
		IRenderOuterTableControl
		IUpdatePanel
		IBorderPaddingControl
		WebControls.
			WizardSideBarListControlItemEventArgs
			IWizardSideBarListControl
			WizardSideBarListControlItem
			IPostBackContainer
	Security.Cryptograpy.
		ICryptoService
	ModelBinding.
		DictionaryHelpers
		ValueProviderResult
		ModelState
		ModelStateDictionary
		ModelBindingExecutionContext
		IValueProvider
		IValueProviderSource
		IUnvalidatedValueProviderSource
	Configuration.
		IServerConfig2
		
[System.Web.ApplicationServices]
System.Web.ApplicationServices.
	Configuration.
		MembershipPasswordCompatibilityMode
	Security.
		IMembershipAdapter
		MembershipCreateStatus
		MembershipCreateUserException
		MembershipPasswordException
		MembershipPasswordFormat
		MembershipProviderCollection
		MembershipUserCollection
		MembershipValidatePasswordEventHandler
		ValidatePasswordEventArgs
```

### CODE REVIEW

**Comment by kasthack** [comment]
bump

File: mcs/class/System.Web.ApplicationServices/System.Web.ApplicationServices.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Can_not_use_encrypted_passwords_with_autogen_keys=You must specify a non-autogenerated machine key to store passwords in the encrypted format. Either specify a different passwordFormat, or change the machineKey configuration to use a non-autogenerated decryption key.
```
Comment (akoeplinger) [inline mcs/class/System.Web.ApplicationServices/System.Web.ApplicationServices.txt]: where does this file come from?

Comment (kasthack) [inline mcs/class/System.Web.ApplicationServices/System.Web.ApplicationServices.txt]: @akoeplinger https://github.com/mono/referencesource/blob/mono/System.Web.ApplicationServices/ApplicationServicesStrings.Designer.cs#L64

Comment (kasthack) [inline mcs/class/System.Web.ApplicationServices/System.Web.ApplicationServices.txt]: @akoeplinger misread 'file' as 'line'.
Property names + values from documentation comments from [ApplicationServicesStrings](https://github.com/mono/referencesource/blob/mono/System.Web.ApplicationServices/ApplicationServicesStrings.Designer.cs#L64) from reference source

**Pull request merged**